### PR TITLE
EPMEDU-1061 Show placeholder text when there are no favourite points

### PR DESCRIPTION
**Pull request by ArtemKleinschmidt**
Show placeholder text when there are no favourite points
Link: https://jira.epam.com/jira/browse/EPMEDU-1061

### CODE REVIEW

File: feature/tabsflow/favourites/src/main/java/com/epmedu/animeal/favourites/FavouritesScreenUI.kt [hunk 1]
```diff
@@ -40,25 +38,54 @@ internal fun FavouritesScreenUI(
...
     ) { padding ->
-        Column(
-            modifier = Modifier
-                .padding(padding)
-                .padding(top = 12.dp, bottom = 32.dp)
-        ) {
-            LazyColumn(
-                verticalArrangement = Arrangement.spacedBy(12.dp),
-                contentPadding = PaddingValues(30.dp)
+        if (state.favourites.isEmpty()) {
```
Comment (Fileek) [inline feature/tabsflow/favourites/src/main/java/com/epmedu/animeal/favourites/FavouritesScreenUI.kt]: Let's use when, if we have else statement

File: feature/tabsflow/favourites/src/main/java/com/epmedu/animeal/favourites/FavouritesScreenUI.kt [hunk 2]
```diff
@@ -40,25 +38,54 @@ internal fun FavouritesScreenUI(
...
+            modifier = Modifier.align(Alignment.Center),
+            textAlign = TextAlign.Center,
+        )
+    }
+}
+
+@Composable
+private fun FavouritesList(
+    padding: PaddingValues,
+    state: FavouritesState,
```
Comment (Fileek) [inline feature/tabsflow/favourites/src/main/java/com/epmedu/animeal/favourites/FavouritesScreenUI.kt]: I assume we won't need the whole state object in this composable, just the list of FavouriteFeedingPoint

File: feature/tabsflow/favourites/src/main/java/com/epmedu/animeal/favourites/FavouritesScreenUI.kt [hunk 3]
```diff
@@ -79,4 +106,16 @@ private fun MoreScreenPreview() {
             )
         ) {}
     }
+}
+
+@AnimealPreview
+@Composable
+private fun MoreScreenEmptyPreview() {
```
Comment (Fileek) [inline feature/tabsflow/favourites/src/main/java/com/epmedu/animeal/favourites/FavouritesScreenUI.kt]: Can you rename it to FavouritesScreenEmptyPreview and the preview above to FavouritesScreenPreview? We missed it somehow

**Comment by ArtemKleinschmidt** [comment]
Resolved comments 
Video is added 
[device-2022-11-02-111131.webm](https://user-images.githubusercontent.com/32295746/199453824-3621d4bf-7403-4f93-a1de-a7fcb5491c99.webm)

**Review by Fileek — APPROVED** [review]

**Review by egorikftp — APPROVED** [review]

**Pull request merged**